packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -410,7 +441,7 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
       const selectedSub: SubscriptionInfo = {
         subscriptionId: AzureAccountManager.subscriptionId,
         tenantId: AzureAccountManager.tenantId!,
-        subscriptionName: "",
+        subscriptionName: AzureAccountManager.subscriptionName ?? "",
```

Comment:
In which case the subsId exists but name not?

Reply:
Old project solution only save tenantid and subscriptionId.